pkg/policy/rule_test.go

```diff
@@ -550,12 +550,12 @@ func (ds *PolicyTestSuite) TestL3Policy(c *C) {
 	expected.Ingress.IPv6Changed = true
 	expected.Ingress.IPv6Count = 2
 	expected.Egress.Map["10.1.0.0/16"] = net.IPNet{IP: []byte{10, 1, 0, 0}, Mask: []byte{255, 255, 0, 0}}
-	expected.Egress.Map["10.128.0.0/9"] = net.IPNet{IP: []byte{10, 128, 0, 0}, Mask: []byte{255, 128, 0, 0}}
-	expected.Egress.Map["10.0.0.0/10"] = net.IPNet{IP: []byte{10, 0, 0, 0}, Mask: []byte{255, 192, 0, 0}}
-	expected.Egress.Map["10.64.0.0/11"] = net.IPNet{IP: []byte{10, 64, 0, 0}, Mask: []byte{255, 224, 0, 0}}
-	expected.Egress.Map["10.112.0.0/12"] = net.IPNet{IP: []byte{10, 112, 0, 0}, Mask: []byte{255, 240, 0, 0}}
+	expected.Egress.Map["10.128.0.0/9"] = net.IPNet{IP: []byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xff, 0xff, 10, 128, 0, 0}, Mask: []byte{255, 128, 0, 0}}
+	expected.Egress.Map["10.0.0.0/10"] = net.IPNet{IP: []byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xff, 0xff, 10, 0, 0, 0}, Mask: []byte{255, 192, 0, 0}}
+	expected.Egress.Map["10.64.0.0/11"] = net.IPNet{IP: []byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xff, 0xff, 10, 64, 0, 0}, Mask: []byte{255, 224, 0, 0}}
+	expected.Egress.Map["10.112.0.0/12"] = net.IPNet{IP: []byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xff, 0xff, 10, 112, 0, 0}, Mask: []byte{255, 240, 0, 0}}
 	expected.Egress.IPv4Changed = true
-	expected.Egress.IPv4Count = 5
+	expected.Egress.IPv4Count = 4
```

Comment:
Why does the expected egress IPv4 count go to 4?

Reply:
This is because the CIDRs are coalesced (10.1.0.0/16 is contained within one of the other CIDRs).

Reply:
OK.

Reply:
Can you add that in a comment?

Also, you can create a new IPv4 with `net.IPv4()`